ld/hc32f46x_param.ld

```diff
@@ -106,8 +106,9 @@ SECTIONS
         /* ICG: 0x400 - 0x41F*/
         .icg_sec :
         {
+            ASSERT(. == (ORIGIN(FLASH) + 0x400), "icg_sec alignment issue");
             KEEP(*(.icg_sec));
-            . = 0x420;
```

Comment:
removes unnessecary padding, which is good but should be documented in the PR desc. 

+ requires patch file to be updated